examples/nlp/language_modeling/megatron_change_num_partitions.py

```diff
@@ -0,0 +1,188 @@
+# Copyright (c) 2021, NVIDIA CORPORATION.  All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+from argparse import ArgumentParser
+
+import torch
+from pytorch_lightning import Trainer
+
+from nemo.collections.nlp.parts.nlp_overrides import NLPDDPPlugin, NLPSaveRestoreConnector
+from nemo.utils import logging, model_utils
+from nemo.utils.app_state import AppState
+
+
+"""
+Usage:
+python megatron_change_num_partitions.py \
+    --model_file=PATH_TO_SRC_FILE \
+    --target_file=PATH_TO_TGT_FILE \
+    --tensor_model_parallel_size=2 \
+    --target_tensor_model_parallel_size=1
+"""
```

Comment:
Could you also add here what models are supported?

Reply:
Added details in the argparse help